### ISSUE TITLE
Reorganize global styles into packages

### ISSUE THREAD

**Issue by tarikhamilton** [masonitedoors/MDL#150]
For Lerna to work properly, the packages should be dependencies of each other, rather than sharing stuff at root. Right now, if I were to make a change to a component (`/components`), that change would not be detected by Lerna when running `lerna publish` because `/packages/external-ui-react` only imports from `/components` (or in the TS update, copies them to build) and nothing else. This has never really been a problem because we usually update the CHANGELOG or add the import for the first time, thus creating a detectable change.

Basically, we aren't utilizing one of Lerna's prime features correctly. Our packages can still share common code as they do today, but they will have to live somewhere in the packages directory. Then Lerna can see if a change another package is depending on. I don't believe all packages have to be published either, so you can think of these as a bunch of little modules linked together rather than modules that pull from the same source. Not super high priority, but wanted to make this issue for visibility and to explain the problem.
